content/post/analytics_brain_trust.md

```diff
@@ -0,0 +1,86 @@
+---
+author: Scott Breitenother
+title: "Building your analytics brain trust"
+cover: "/img/analytics_brain_trust.jpg"
+publishDate: "2018-08-05"
+draft: true
+tags: 
+ - learning
+ - networking
+ - analytics
+ - brain trust
+---
+
+Imagine you hit a roadblock while trying to tackle a complex piece of analysis, using a python function or designing your first data organization. What do you do? Of course you first search the internet, but what do you do when you're really stuck? I like to phone a friend. 
+
+In this post I explore my favorite learning style - learning from others - and the steps to building your own analytics brain trust. I have used this approach for many challenges (including building an Analytics team from the ground up) and I believe it can be applied to most challenges.
+<!--more-->
+
+Before I dive in, it is worth noting that you can only be successful at this if you truly enjoy talking to and learning from other people.
+
+
+## Origin story
+
+In a past life I worked as a management consultant, often undertaking strategy projects in obscure industries (fun fact: if you can think of it, a company somewhere in the world does it). Due to the niche nature of these industries, internet resources were typically unhelpful or unavailable. As a result, we often spoke to experts in the field - scientists, business leaders, researchers and end-users.
```

Comment:
Strike the parenthetical in the first sentence, or move it to a footnote and add an example.

Reply:
🚨 PEDANT ALERT 🚨

Another place where we need an em-dash :)

http://www.thepunctuationguide.com/em-dash.html

> we often spoke to experts in the field — scientists, business leaders, researchers and end-users.

NOTE: that guide recommends using em-dashes without spaces, but I prefer the AP style guide recommendation of adding a whitespace on either side, as in this example.